src/Json/JsonFile.php

```
@@ -18,15 +35,20 @@ final class JsonFile extends \Composer\Json\JsonFile
-    public static function encode(mixed $data, int $options = 448, string $indent = self::INDENT_DEFAULT): string
-    {
-        $result = parent::encode($data, $options);
+    private array $mapKeys = [];
 
-        return JsonFormatter::format($result, self::$encodeArrayKeys, self::$encodeIndent, false);
+    private bool $parsed = false;
+
+    public static function encode(
+        mixed $data,
+        int $options = self::DEFAULT_OPTIONS,
+        string $indent = self::INDENT_DEFAULT,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$indent'. (undefined)

(UnusedFormalParameter)